src/sagas/publicChannels/publicChannels.selectors.test.ts

```diff
@@ -103,46 +161,143 @@ describe('publicChannelsSelectors', () => {
+          "id": "5",
+          "message": "message5",
+          "pubKey": "BPeZAjdqR+hMPeL3hklkyZnZFmzAR5DRpbSiD79W7NQlBuVVD5R0fqS4oWOrsFO12NhMxII73B8+i0Ffzz6nYq4=",
+          "signature": "",
+          "type": 1,
+        },
+        Object {
+          "channelId": "",
+          "createdAt": 1417155108,
+          "id": "6",
+          "message": "message6",
+          "pubKey": "BPeZAjdqR+hMPeL3hklkyZnZFmzAR5DRpbSiD79W7NQlBuVVD5R0fqS4oWOrsFO12NhMxII73B8+i0Ffzz6nYq4=",
+          "signature": "",
+          "type": 1,
+        },
+        Object {
+          "channelId": "",
+          "createdAt": 1417155120,
+          "id": "7",
+          "message": "message7",
+          "pubKey": "BCNYTCbhFS8UirgYAQlVjn3MaqJQtxdK8b5h4X/HAoZsSspgtdly990/S0U3zpb3WCiXJaEYt7DYgJ97Ccow/L4=",
+          "signature": "",
+          "type": 1,
+        },
+      ]
+    `);
+  });
+
+  it('get grouped messages', async () => {
+    const messages = publicChannelsSelectors.currentChannelMessagesGroupedByDay(
+      store.getState()
+    );
+
+    expect(messages).toMatchInlineSnapshot(`
 Array [
   Object {
-    "channelId": "",
-    "createdAt": 0,
-    "id": "1",
-    "message": "message1",
-    "pubKey": "12",
-    "signature": "",
-    "type": 1,
-  },
-  Object {
-    "channelId": "",
-    "createdAt": 0,
-    "id": "0",
-    "message": "message0",
-    "pubKey": "12",
-    "signature": "",
-    "type": 1,
+    "day": "Today",
+    "messages": Array [
+      Object {
+        "createdAt": "1:54 PM",
+        "id": "1",
+        "message": "message1",
+        "nickname": "userName",
+        "type": 1,
+      },
+    ],
   },
   Object {
-    "channelId": "",
-    "createdAt": 0,
-    "id": "2",
-    "message": "message2",
-    "pubKey": "12",
-    "signature": "",
-    "type": 1,
+    "day": "Nov 17",
+    "messages": Array [
+      Object {
+        "createdAt": " 11:57 AM",
+        "id": "0",
+        "message": "message0",
+        "nickname": "userName",
+        "type": 1,
+      },
+    ],
   },
   Object {
-    "channelId": "",
-    "createdAt": 0,
-    "id": "4",
-    "message": "message4",
-    "pubKey": "12",
-    "signature": "",
-    "type": 1,
+    "day": "Nov 28",
+    "messages": Array [
+      Object {
+        "createdAt": " 6:12 AM",
+        "id": "4",
+        "message": "message4",
+        "nickname": "userName",
+        "type": 1,
+      },
+      Object {
+        "createdAt": " 6:12 AM",
+        "id": "7",
+        "message": "message7",
+        "nickname": "userName2",
+        "type": 1,
+      },
+      Object {
+        "createdAt": " 6:11 AM",
+        "id": "6",
+        "message": "message6
+message2",
+        "nickname": "userName",
+        "type": 1,
+      },
+      Object {
+        "createdAt": " 5:53 AM",
+        "id": "5",
+        "message": "message5",
+        "nickname": "userName",
+        "type": 1,
+      },
+    ],
   },
 ]
 `);
   });
 });
 
 export {};
+// TODO wiadomosci od innych uzytkownikow
```

Comment:
These should be in English so I can follow

Reply:
yes sure, i missed it